ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java

```diff
@@ -66,7 +64,9 @@ public void contactInfoTest() throws Exception {
         activeVisitsPage.search(patient.identifier);
         patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         patientDashboardPage.clickOnShowContact();
-        assertTrue(patientDashboardPage.getTelephoneNumber().contains("111111111"));
+        patientDashboardPage.waitForPageToLoad();
+        assertTrue(patientDashboardPage.getTelephoneNumber().contains("111111111"));   
```

Comment:
You need to configure your IDE to only track your changes otherwise the `assertTrue(patientDashboardPage.getTelephoneNumber().contains("111111111"));`   is not changes at all

Reply:
This is actually the fault of Git and common to how it does patches. The `assertTrue()` line isn't new, but it's surrounded by two lines that are new, so Git treats this as though it were the delete of a single line and the insertion of 3 lines (which makes the merging logic easier).